pysmiles/smiles_helper.py

```diff
@@ -142,7 +142,7 @@ def format_atom(molecule, node_key, default_element='*'):
         name = name.lower()
 
     if (stereo is None and isotope == '' and charge == 0 and default_h and
-            class_ == '' and name.lower() in 'b c n o p s se as *'.split()):
+            class_ == '' and name.lower() in 'b c n o p s *'.split()):
```

Comment:
I think this looks good, thank you! One comment I have is that additionally to the elements mentioned here, according to Wikipedia, F, Cl, Br and I are also part of the organic subset and can also be written without square brackets. This is also implemented in the pysmiles reader. However, this would have to be implemented a little differently, because while F, Cl, Br and I can appear without square brackets, they cannot appear as lowercase letters in aromatic rings, unlike 'b c n o p s'. So, I think it would also work to replace the line with something like:
`... and (name.lower() in 'b c n o p s *'.split() or name in 'F Cl Br I'.split())`
I think this should also work in the pysmiles reader. However, it is not necessary since it is just an abbreviation, so I can understand if for safety reasons you prefer to keep it as is now.

Reply:
Excellent catch, fixed!